lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -44,7 +44,24 @@ def test_aspects():
                 ax.plot3D(*zip(start*scale, end*scale))
     for i, ax in enumerate(axs):
         ax.set_box_aspect((3, 4, 5))
-        ax.set_aspect(aspects[i])
+        ax.set_aspect(aspects[i], adjustable='datalim')
+
+
+@mpl3d_image_comparison(['aspects_adjust_box.png'], remove_text=False)
+def test_aspects_adjust_box():
+    aspects = ('auto', 'equal', 'equalxy', 'equalyz', 'equalxz')
+    fig, axs = plt.subplots(1, len(aspects), subplot_kw={'projection': '3d'})
```

Comment:
```suggestion
    fig, axs = plt.subplots(1, len(aspects), fig_kw = {'figsize': (10, 3)}, subplot_kw={'projection': '3d'})
```
or something to make the figure wider and lower. One reason is to avoid tick labels being cropped.